src/js/contexts/DataContext/index.d.ts

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { View } from '../../components/Data';
+
+interface DataContextType {
+  data: any[];
```

Comment:
Will this always be an array of objects or could it be other types?

Reply:
I had it as this since this was the prop in the code
```
  data: dataProp = [],
```

Reply:
Seems like it is valid for it to be either an empty array or an array of objects
<img width="1190" alt="Screenshot 2025-03-03 at 2 45 49 PM" src="https://github.com/user-attachments/assets/060d4696-128c-435f-a893-301784a7c624" />

Reply:
If there isn't a good way to set the type for that then we can leave as `data: any[];`

Reply:
let me know I changed it to the following:

```
  data: Array<{}> | [];
```